stock_move_consolidated/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_stock_move_consolidation,access_stock_move_consolidation,model_stock_move_consolidation,base.group_user,1,1,1,1
```

Comment:
We need to be careful that this doesnt make it to a production server, it gives everyone a right to consolidate stock moves.

Reply:
the group that gets access to this should be the people who are in the inventory administrator group

Reply:
Yeah, that make sense